front-end-admin/src/services/helpers.js

```diff
@@ -1,26 +1,45 @@
 // export const URL = 'http://localhost:8800/';
 export const URL = 'http://localhost:8080/donation-api';
 export const getJSON = apiRoute =>
-  fetch(`${URL}/${apiRoute}`, { method: 'GET' }).then(res => res.json());
+  fetch(`${URL}/${apiRoute}`, {
+    method: 'GET'
+  }).then(res => res.json());
+
 export const postJSON = (apiRoute, data) =>
   fetch(`${URL}/${apiRoute}`, {
     method: 'POST',
     body: JSON.stringify(data),
     headers: new Headers({
-      'Content-Type': 'application/json'
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Origin': '*',
+      'Access-Control-Allow-Methods': 'POST',
+      'Access-Control-Allow-Headers': 'Origin'
     }),
     mode: 'cors',
     cache: 'default'
-  }).then(res => res.json());
+  })
+    .then(res => res.json())
+    .catch(error => console.error('Error:', error))
+    .then(response => console.log('Success:', response));
+
 export const patchJSON = (apiRoute, data) =>
   fetch(`${URL}/${apiRoute}`, {
     method: 'PATCH',
     body: JSON.stringify(data),
     headers: new Headers({
-      'Content-Type': 'application/json'
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Origin': 'http://localhost:8081',
+      'Access-Control-Allow-Methods': 'PATCH',
+      'Access-Control-Allow-Headers': 'Origin'
```

Comment:
This looked like a good lead to solving the CORS issue, although I couldn't get it work.. https://stackoverflow.com/questions/35315090/spring-boot-enable-global-cors-support-issue-only-get-is-working-post-put-and